src/main/java/net/gnomeffinway/depenizen/objects/MythicMobs/MythicMobsMob.java

```diff
@@ -0,0 +1,287 @@
+package net.gnomeffinway.depenizen.objects.MythicMobs;
+
+import net.aufdemrand.denizen.objects.dEntity;
+import net.aufdemrand.denizencore.objects.*;
+import net.aufdemrand.denizencore.tags.Attribute;
+import net.aufdemrand.denizencore.tags.TagContext;
+import net.aufdemrand.denizencore.utilities.debugging.dB;
+import net.elseland.xikage.MythicLib.Adapters.AbstractEntity;
+import net.elseland.xikage.MythicLib.Adapters.Bukkit.BukkitEntity;
+import net.elseland.xikage.MythicMobs.API.IMobsAPI;
+import net.elseland.xikage.MythicMobs.Mobs.ActiveMob;
+import net.elseland.xikage.MythicMobs.MythicMobs;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+
+import java.util.UUID;
+
+public class MythicMobsMob implements dObject, Adjustable {
+
+    public static MythicMobsMob valueOf(String uuid) {
+        return valueOf(uuid, null);
+    }
+
+    @Fetchable("mythicmob")
+    public static MythicMobsMob valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+        try {
+            string = string.replace("e@", "").replace("mythicmob@", "");
+            UUID uuid = UUID.fromString(string);
+            IMobsAPI api = MythicMobs.inst().getAPI().getMobAPI();
```

Comment:
Either make this a static variable too or if that doesn't work, make the other one consistent with this